resources/public/api/conf/1.0/createOrAmendCodingOutUnderpayments.yaml

```diff
@@ -3,21 +3,11 @@
     This endpoint enables you to overwrite coding out underpayments and debt amounts held by HMRC for a previous tax year and given National Insurance number, and to amend previously submitted overwrite amounts. This endpoint can only be used after the tax year has ended.
     
     ### Test data
-    <p>Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.</p>
-    <table>
-        <thead>
-            <tr>
-                <th>Header Value (Gov-Test-Scenario)</th>
-                <th>Scenario</th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr>
-                <td><p>N/A - DEFAULT</p></td>
-                <td><p>Simulate a successful response with a self-assessment accounts.</p></td> 
-            </tr>
-        </tbody>
-    </table>
+    Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+    
+    | Header Value (Gov-Test-Scenario)      | Scenario                                                         |
+    |---------------------------------------|------------------------------------------------------------------|
+    | N/A - DEFAULT                         | Simulates a successful response with a self-assessment accounts. |    
```

Comment:
I know you haven't changed the wording but i think we should confirm if this is correct as it says `a self-assessment accounts` which doesn't really make sense? Perhaps Ben can confirm

Reply:
Seems to be a recurring sentence too

Reply:
There seems to be inconsistency in the language and types of gov-test-scenarios mentioned in the tech spec and the OAS. Since this ticket only involved conversion of gov-test-scenario tables from html to md, crosschecking these details for each endpoint is outside its scope. Another JIRA ticket should be created for this purpose.